Sources/HaCWebsiteLib/ViewModels/NotFound.swift

```diff
@@ -0,0 +1,27 @@
+import HaCTML
+
+struct NotFound {
+  
+  var node: Node {
+    return UI.Pages.base(
+      title: "Page Not Found",
+      content: Fragment(
+        El.Div[Attr.className => "NotFound"].containing(
+          El.Img[
+            Attr.className => "LandingIntroduction__titleImage",
```

Comment:
It looks like we still have `LandingIntroduction__titleImage` and `LandingIntroduction__tagLine` here?

Reply:
My thought was that the 404 page styling is dependent on how you style the logo and strapline in general, but I wasn't sure how you wanted that styling separated from the rest of the landing page styling.

Reply:
I agree - it makes sense that this construct is consistent where we use it on the site. This leads to our naming being incorrect in this case, if it's used in more places than our landing page.

Do you think that the same approach you used for the button would make sense here? Pulling it out into some kind of `SiteLogo` component?

Reply:
Ok yeah I'll do that now